app/dashboard/utils.py

```diff
@@ -998,3 +994,38 @@ def list_urls(lis, acc=None):
 
 def get_custom_avatars(profile):
     return CustomAvatar.objects.filter(profile=profile).order_by('-id')
+
+
+
+
+
+def getTransaction__(txid):
+    """Returns a transaction"""
+    try:
+        return web3.eth.getTransaction(txid)
+    except TransactionNotFound:
+        return None
+
+
+def getBlock__(latest):
```

Comment:
I don't understand the naming convention here - what does the trailing double underscore signify?

Reply:
Regarding name conversion, there's nothing special with it. I added the trailing underscore to differentiate it from the original function when I was testing it, I then copy pasted it here.

Reply:
I will remove the underscore